README.md

```diff
@@ -221,6 +223,40 @@ wavefront_sender = WavefrontProxyClient(
 | `distribution_port` | `histogramDistListenerPorts=` |
 | `tracing_port` | `traceListenerPorts=` |
 
+### Option 3: Create a WavefrontClient
+Use `WavefrontClientFactory` to create a `WavefrontClient` instance, which can send data directly to a Wavefront service or send data using a Wavefront Proxy.
+
+The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` to send multiple Wavefront services.
```

Comment:
```suggestion
The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` to send to multiple Wavefront services.
```
Shouldn't we use `endpoints` instead of `services` here?

Reply:
Updated the sentence:
The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` instance, which can send data to multiple Wavefront services.